### ISSUE TITLE
name 'ser' is not defined. Did you mean: 'set'?

### ISSUE THREAD

**Issue by tdjastrzebski** [N0ury/dmm_util#17]
```
C:\Temp>python -m fluke_28x_dmm_util -p com6 recordings list
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\__main__.py", line 16, in <module>
    cli()
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\__main__.py", line 13, in cli
    dmm_util.main()
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\dmm_util.py", line 629, in main
    do_recordings(args.command[1:])
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\dmm_util.py", line 449, in do_recordings
    nb_recordings = int(qsls()['nb_recordings'])
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\dmm_util.py", line 138, in qsls
    res = meter_command("qsls")
  File "C:\Users\Thomas\AppData\Roaming\Python\Python310\site-packages\fluke_28x_dmm_util\dmm_util.py", line 579, in meter_command
    ser.write(cmd.encode()+b'\r')
NameError: name 'ser' is not defined. Did you mean: 'set'?
```

**Comment by N0ury** [comment]
Same here.
Please wait for next commit.

**Comment by N0ury** [comment]
Should be solved in commit [c92b1b1](https://github.com/N0ury/dmm_util/commit/c92b1b16a1d5e2593059ce53a13940256d3018d4)
